verifying/verifying.go

```diff
@@ -52,7 +52,7 @@ func VerifyVRFNonce(nonce *uint64, m *shared.VRFNonceMetadata, opts ...OptionFun
 	}
 
 	if res.Nonce == nil || *res.Nonce != *nonce {
-		return fmt.Errorf("nonce %v is not valid for node %v", *nonce, m.NodeId)
+		return fmt.Errorf("nonce %v is not valid for node %x", *nonce, m.NodeId)
```

Comment:
`types.NodeID` has a `String()` method that prints the nodeID in short form.

```suggestion
		return fmt.Errorf("nonce %v is not valid for node %s", *nonce, m.NodeId)
```